pkg/controller/policyrecommendation/policyrecommendation_controller.go

```diff
@@ -124,6 +124,7 @@ func Add(mgr manager.Manager, opts options.AddOptions) error {
 			certificatemanagement.CASecretName,
 			render.ManagerInternalTLSSecretName,
 			render.TigeraLinseedSecret,
+			render.PolicyRecommendationTLSSecretName,
```

Comment:
We don't need to watch the PolicyRecommendationTLS Secret anymore? How come?

Reply:
Actually, I am adding the watch not removing it.